viewing_party/party.py

```diff
@@ -1,23 +1,127 @@
 # ------------- WAVE 1 --------------------
 
+from typing import Counter
+
+
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if bool(title) == True and bool(genre) == True and bool(rating) == True:

```

Comment:
You don't need to convert to `bool` or convert to `True`:

```suggestion
    if title and genre and rating:
```